prime-router/src/main/kotlin/cli/LivdTableDownload.kt

```diff
@@ -29,38 +40,42 @@ class LivdTableDownload() : CliktCommand(
 	If the file is found, it downloads the file into the directory specified by the --output-dir <path> option.
 	If the option is not specified, it will download the file to ./build directory.
         
-    Example:
-      The following command will download the latest LIVD-SARS-CoV-2-yyyy-MM-dd.xlsx from the above URL.  It will 
-      store the file under the ./junk directory.
+    Example: The following command will download the latest LIVD-SARS-CoV-2-yyyy-MM-dd.xlsx from the above URL.  
+      It will store the LIVD-SARS-CoV-2-xyz.csv file under the ./junk directory.
       
-         ./prime livd-table --output-dir ./junk 
+         ./prime livd-table-download --output-file ./junk/LIVD-SARS-CoV-2-xyz.csv
     """
 ) {
     private val defaultOutputDir = "./build"
-    private val outputDir by option(
-        "--output-dir",
-        metavar = "<path>",
-        help = "interpret `--output` relative to this directory (default: \"$defaultOutputDir\")"
-    ).default(defaultOutputDir)
+    private val outputDirFile by option(
+        "--output-file",
+        metavar = "<path/filename>",
+        help = "Directory/Filename for the CSV output file."
```

Comment:
Be more descriptive with these help strings.  E.g. Output pathname where to store the LIVD data as CSV.  Also, this needs to be file and not directory as you names the option output-file, so change the variables too.  Choose a dir or a filename, not both.

Reply:
Done.